diff --git a/dist/index.js b/dist/index.js
index 139496791435e05986df46357345f4d26e63e875..b1a7164019acaa701295531cab7920a94de2e6aa 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -799,15 +799,6 @@ var SelectViewport = /* @__PURE__ */ React.forwardRef(
     const composedRefs = (0, import_react_compose_refs.useComposedRefs)(forwardedRef, contentContext.onViewportChange);
     const prevScrollTopRef = React.useRef(0);
     return /* @__PURE__ */ (0, import_jsx_runtime.jsxs)(import_jsx_runtime.Fragment, { children: [
-      /* @__PURE__ */ (0, import_jsx_runtime.jsx)(
-        "style",
-        {
-          dangerouslySetInnerHTML: {
-            __html: `[data-radix-select-viewport]{scrollbar-width:none;-ms-overflow-style:none;-webkit-overflow-scrolling:touch;}[data-radix-select-viewport]::-webkit-scrollbar{display:none}`
-          },
-          nonce
-        }
-      ),
       /* @__PURE__ */ (0, import_jsx_runtime.jsx)(Collection.Slot, { scope: __scopeSelect, children: /* @__PURE__ */ (0, import_jsx_runtime.jsx)(
         import_react_primitive.Primitive.div,
         {
diff --git a/dist/index.mjs b/dist/index.mjs
index df29cd391959495b6c6413831bcba6e76a4b930d..b69fce8bd314ebde16ecbfb4013cf71c95ebbe36 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -729,15 +729,6 @@ var SelectViewport = /* @__PURE__ */ React.forwardRef(
     const composedRefs = useComposedRefs(forwardedRef, contentContext.onViewportChange);
     const prevScrollTopRef = React.useRef(0);
     return /* @__PURE__ */ jsxs(Fragment2, { children: [
-      /* @__PURE__ */ jsx(
-        "style",
-        {
-          dangerouslySetInnerHTML: {
-            __html: `[data-radix-select-viewport]{scrollbar-width:none;-ms-overflow-style:none;-webkit-overflow-scrolling:touch;}[data-radix-select-viewport]::-webkit-scrollbar{display:none}`
-          },
-          nonce
-        }
-      ),
       /* @__PURE__ */ jsx(Collection.Slot, { scope: __scopeSelect, children: /* @__PURE__ */ jsx(
         Primitive.div,
         {
